extensions/cli/sentinel2/src/test/java/mil/nga/giat/geowave/format/sentinel2/Tests.java

```diff
@@ -0,0 +1,225 @@
+/*******************************************************************************
+ * Copyright (c) 2013-2018 Contributors to the Eclipse Foundation
+ * 
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License,
+ * Version 2.0 which accompanies this distribution and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0.txt
+ ******************************************************************************/
+package mil.nga.giat.geowave.format.sentinel2;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.text.ParseException;
+import java.util.Date;
+
+import javax.ws.rs.core.HttpHeaders;
+
+import org.apache.commons.io.IOUtils;
+import org.geotools.coverageio.gdal.jp2ecw.JP2ECWReader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import mil.nga.giat.geowave.adapter.vector.utils.DateUtilities;
+
+public class Tests
+{
+	private final static Logger LOGGER = LoggerFactory.getLogger(Tests.class);
+
+	// use the same workspace directory as the ITs to consolidate what is
+	// downloaded
+	public static final String WORKSPACE_DIR = "../../../test/sentinel2";
+
+	// Default authentication settings filename for Theia
+	public static final String THEIA_AUTHENTICATION_FILE = "auth_theia.txt";
+
+	// URL of raster to validate JP2 support in GDAL.
+	private final static String JP2ECW_RASTER_TEST_URL = "http://sentinel-s2-l1c.s3.amazonaws.com/tiles/30/T/WN/2017/10/16/0/preview/B01.jp2";
```

Comment:
it looks like the tests are already setup to download the file, can we just remove the file from the baseline and have it automatically downloaded?  It would be easier for IP review, and seems to already be handled here.

Reply:
Thanks a lot for your advices and for reviewing. I have added a new JP2 in data folder to check when testing the JP2ECW plugin. I have verified that no all JP2 files from sentinel2 get to crash the GDAL 1.9.2 redist for Linux. This file is little and breaks GDAL. My apologies I don't remember whether I downloaded it from AWS or PEPS (LEVEL1C mirror at France).